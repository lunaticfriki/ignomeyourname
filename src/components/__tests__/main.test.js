import React from 'react'
import renderer from 'react-test-renderer'

import Main from '../main'

describe('Main component suite', () => {
  test('Main snapshot', () => {
    const component = renderer.create(<Main />)
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
