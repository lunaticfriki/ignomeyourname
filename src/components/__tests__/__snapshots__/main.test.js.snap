// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main component suite Main snapshot 1`] = `
<div
  className="sc-AxjAm eVkhsz"
>
  <div
    className="sc-AxirZ kviWPR"
  >
    <form
      onSubmit={[Function]}
    >
      <label
        className="c-use-gnome"
        htmlFor="use-gnome-findyour gnome"
      >
        Find your gnome
        <select
          data-testid="use-gnome-findyour gnome"
          disabled={true}
          id="use-gnome-findyour gnome"
          onBlur={[Function]}
          onChange={[Function]}
          value=""
        >
          <option
            hidden={true}
          >
             Choose your gnome!
          </option>
        </select>
      </label>
      <button>
        Search!
      </button>
    </form>
  </div>
  <div
    className="sc-AxiKw hlbvwP"
  >
    <div
      className="c-no-info"
    >
      <p>
        Please, select a gnome
      </p>
      <a
        href="https://unsplash.com/photos/xXs6qC7yBao"
        target="blank"
      >
        <img
          alt="Please, select a gnome"
          src="anton-vorobiev.jpg"
        />
      </a>
      <i>
        Photo by Anton Vorobiev
      </i>
    </div>
  </div>
</div>
`;
